### ISSUE TITLE
World generation parameters should be adjustable at the new world screen

### ISSUE THREAD

**Issue by TheLoneWolfling** [BrainSlugs83/NewBiospheresMod#12]
Currently, to set up a new world, you have to exit the game, edit the config file, reboot the game, and start the world.

It would be wonderful if there was a way to edit the config for a new world at the world creation screen, like with the superflat world currently.

Bonus points if you can save / load presets.

**Comment by BrainSlugs83** [comment]
I agree, that would be a pretty cool usability enhancement! -- Do you know of any other forge mods that can do this? 

(Might jump start my research if there's a good example of how to implement this in another mod that already does something similar. -- Otherwise, I just have to figure it out from scratch.)

**Comment by kvaw** [comment]
[Ted's World Gen Mods](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1281910-teds-world-gen-mods-realistic-world-gen-alpha-1-3) all are customizable from the world select screen
